packages/host/app/components/matrix/register-user.gts

```diff
@@ -518,7 +529,10 @@ export default class RegisterUser extends Component<Signature> {
       email: this.email,
       name: this.name,
     };
-    this.validateEmail.perform();
+    this.validateEmail.perform().catch((e) => {
+      console.log('Error registering', e);
+      this.formError = `There was an error registering: ${e.message}`;
```

Comment:
Are we ok with displaying caught error's message directly, in an unfiltered way? I suppose these can get too technical/cryptic for the average user to see.

Reply:
unsure, this is something I struggle with regularly. The message when you fail to enter an email address is `Unable to parse email address` which should be understandable to most. Even the `Registration is disabled` is hopefully intelligible in the first screenshot, surrounded by confusing noise.

Either you add specific handling for each known error and then a generic fallback for unknown ones or you just expose the message directly. I don’t know how to choose!